seacrowd/sea_datasets/mswc/mswc.py

```diff
@@ -0,0 +1,308 @@
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+@dataclass
+class SMSASeacrowdConfig(SEACrowdConfig):
+    """BuilderConfig for Nusantara."""
+
+    language: str = None
+    audio_format: str = None
+
+
+class MSWC(datasets.GeneratorBasedBuilder):
+    """
+    Multilingual Spoken Words Corpus is a large and growing audio dataset of spoken words in 50 languages collectively spoken by over 5 billion people, for academic research and commercial applications in keyword spotting and spoken term search.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = []
+
+    for language in _LANGUAGES:
+        for format in _FORMATS:
+            subset_id = f"{language}_{format}"
+            BUILDER_CONFIGS.append(
+                SMSASeacrowdConfig(
+                    name=f"{subset_id}_source",
+                    version=SOURCE_VERSION,
+                    description=f"{_DATASETNAME} source schema",
+                    schema="source",
+                    subset_id=subset_id,
+                    language=language,
+                    audio_format=format,
+                ),
+            )
+
+    seacrowd_schema_config: list[SEACrowdConfig] = []
+
+    for seacrowd_schema in _SUPPORTED_SCHEMA_STRINGS:
+        for language in _LANGUAGES:
+            for format in _FORMATS:
+                subset_id = f"{language}_{format}"
+                seacrowd_schema_config.append(
+                    SMSASeacrowdConfig(
+                        name=f"{subset_id}_{seacrowd_schema}",
+                        version=SEACROWD_VERSION,
+                        description=f"{_DATASETNAME} {seacrowd_schema} schema",
+                        schema=f"{seacrowd_schema}",
+                        subset_id=subset_id,
+                        language=language,
+                        audio_format=format,
+                    )
+                )
+
+    BUILDER_CONFIGS.extend(seacrowd_schema_config)
+
+    DEFAULT_CONFIG_NAME = f"{_LANGUAGES[0]}_{_FORMATS[0]}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "file": datasets.Value("string"),
+                    "is_valid": datasets.Value("bool"),
+                    "language": datasets.ClassLabel(num_classes=3),
+                    "speaker_id": datasets.Value("string"),
+                    "gender": datasets.ClassLabel(num_classes=4),
+                    "keyword": datasets.Value("string"),
+                    "audio": datasets.Audio(decode=False, sampling_rate=16000 if self.config.audio_format == "wav" else 48000),
+                }
+            )
+
+        elif self.config.schema == f"seacrowd_{str(TASK_TO_SCHEMA[Tasks.SPEECH_RECOGNITION]).lower()}":
+            features = schemas.speech_text_features
```

Comment:
I guess this can be solved if we remove `sampling_rate=16_000` from [this line](https://github.com/SEACrowd/seacrowd-datahub/blob/647f7c904c2e3a71c1152743be1e1091fe1aa76a/seacrowd/utils/schemas/speech_text.py#L10). What do you think?

cc: @sabilmakbar @SamuelCahyawijaya @ljvmiranda921 @Enliven26

Reply:
Yeah I think removing it will solve the issue, as indicated in this [HF doc](https://huggingface.co/docs/datasets/v2.18.0/en/package_reference/main_classes#datasets.Audio)